deeptools/heatmapper.py

```diff
@@ -786,16 +786,23 @@ def get_regions_and_groups(regions_file, onlyMultiplesOf=1,
         previnterval = None
         duplicates = 0
         totalintervals = 0
+        groupintervals = 0
         includedintervals = 0
         group_labels = []
         group_idx = 0
         bed_file = deeptools.readBed.ReadBed(regions_file)
         for ginterval in bed_file:
-            totalintervals += 1
             if ginterval.line.startswith("track") or ginterval.line.startswith("browser"):
                 continue
+            if ginterval.line.startswith('#') and totalintervals == 0:
+                continue
 
             if ginterval.line.startswith('#'):
+                # check for labels with no associated entries
+                if groupintervals == 0:
```

Comment:
I think this case should contain the previous one on line 797, thus I think that 797 and 798 are not necesary